proto/v1/database_service.proto

```diff
@@ -55,6 +56,35 @@ service DatabaseService {
       get: "/v1/{name=environments/*/instances/*/databases/*/schema}"
     };
   }
+
+  rpc GetDatabaseBackupSetting(GetDatabaseBackupSettingRequest) returns (DatabaseBackupSetting) {
+    option (google.api.http) = {
+      get: "/v1/{name=environments/*/instances/*/databases/*/backupSetting}"
+    };
+  }
+
+  rpc ListDatabaseBackupSetting(ListDatabaseBackupSettingRequest) returns (ListDatabaseBackupSettingResponse) {
```

Comment:
Since backup setting is a single resource, do we need this method? If needed, shall this be called SearchBackupSetting since we're looking for backup settings for databases within a project?